UnityProject/Assets/Scripts/Messages/Server/SpritesMessages/SpriteUpdateMessage.cs

```diff
@@ -203,11 +203,12 @@ public static void SendToAll(Dictionary<SpriteHandler, SpriteHandlerManager.Spri
 		IEnumerable<KeyValuePair<SpriteHandler, SpriteHandlerManager.SpriteChange>> ToSend)
 	{
 		foreach (var VARIABLE in ToSend)
-
 		{
+			if(VARIABLE.Value == null) continue;
+
 			ToReturn.Append(VARIABLE.Key.GetMasterNetID().netId.ToString());
 			ToReturn.Append("@");
-			ToReturn.Append(VARIABLE.Key.name);
+			ToReturn.Append(VARIABLE.Key?.name ?? "default_name");
```

Comment:
NullReferenceException
UnityEngine.Object.get_name () (at /home/builduser/buildslave/unity/build/Runtime/Export/Scripting/UnityEngineObject.bindings.cs:189)
SpriteUpdateMessage.GenerateStates (SpriteUpdateMessage spriteUpdateMessage, System.Collections.Generic.IEnumerable`1[T] ToSend) (at /github/workspace/UnityProject/Assets/Scripts/Messages/Server/SpritesMessages/SpriteUpdateMessage.cs:210)
SpriteUpdateMessage.GenerateMessage (System.Collections.Generic.IEnumerable`1[T] ToSend) (at /github/workspace/UnityProject/Assets/Scripts/Messages/Server/SpritesMessages/SpriteUpdateMessage.cs:197)
SpriteUpdateMessage.SendToAll (System.Collections.Generic.Dictionary`2[TKey,TValue] ToSend) (at /github/workspace/UnityProject/Assets/Scripts/Messages/Server/SpritesMessages/SpriteUpdateMessage.cs:187)
SpriteHandlerManager.UpdateClients () (at /github/workspace/UnityProject/Assets/Scripts/Sprite Handler/SpriteHandlerManager.cs:102)
SpriteHandlerManager.LateUpdate () (at /github/workspace/UnityProject/Assets/Scripts/Sprite Handler/SpriteHandlerManager.cs:88)

Reply:
Here too, it is unconvenient but this should use the regulat ``== null`` expression for null checking.

If you're using Rider, you can see it even shows a warning
![image](https://user-images.githubusercontent.com/43683714/89592145-e5d5d680-d819-11ea-819f-c3f8335684dd.png)